runtime/layers/fpm/Dockerfile

```diff
@@ -7,7 +7,8 @@ COPY bootstrap bootstrap
 COPY php.ini bref/etc/php/conf.d/bref.ini
 COPY php-fpm.conf bref/etc/php-fpm.conf
 
+# Build the final image from the lambci image that is close to the production environment
 FROM lambci/lambda:provided
 
-WORKDIR /
 COPY --from=0  /opt /opt
+WORKDIR /var/task
```

Comment:
@Guillaume-Rossignol I applied the same thing as in the `function` images: set the workdir to `/var/task`.

Reply:
In fact, the workdir un lambci/lambda is `/var/task` so we could juste remove this line